upgrade/Creating_hybrid_configurations.md

```diff
@@ -0,0 +1,30 @@
+# Upgrading management and managed nodes with different CSM versions using IUF
+
+For certain use cases, there may be a need to combine different recipes of CSM to bring up the
+management and managed nodes. For ex: If there is need to use a feature supported for specific
+CPU architecture for managed nodes which is present in a specific recipe of CSM.
+
+## Description
+
+1. Decide the recipes for management and managed nodes. For ex: Recipe **`a.b.c`** for CSM version  
+`1.5.x` for managed nodes and recipe **`x.y.z`** for CSM version **`1.6.x`** for management nodes.
+
+1. Ensure that the CSM cluster is operational with the lowest among the selected recipes for both
+management and managed nodes.
+
+Use the [Upgrade CSM and additional products with IUF](../operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md) procedure to upgrade CSM.
+In the above example, it will be **`a.b.c`** for CSM version **`1.5.x`** for both nodes.
+
+1. Go through the product versions file for the higher version and come up with a set
+of products which have to be upgraded for the recipe. In the above example, the
+products from **`x.y.z`** which should be upgraded for CSM version **`1.6.x`**
+should be selected.
+
+1. Use the [Upgrade CSM and additional products with IUF](../operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md) procedure to upgrade CSM till
+[step 9, Managed rollout](../operations/iuf/workflows/managed_rollout.md), skip section
+"2.2 Compute Nodes". That is, do not upgrade compute nodes to the recipe `x.y.z`
+on CSM `1.6.x` and keep the compute nodes on the `a.b.c` recipe on CSM `1.5.x`.`  
+
+## Post install Testing
+
+Run required tests to ensure that the upgrade works for the intended use cases.
```

Comment:
Could this be more specific? Should the normal post install checks be executed and could a link to the post install checks be added?

Reply:
Changed